domains/pradeep.is-cool.dev

```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Personal Portfolio Website",
+
+  "domain": "is-cool.dev",
+  "subdomain": "pradeep",
+
+  "owner": {
+    "email": "pradeepmalla2001@gmail.com"
+  },
+
+  "record": {
+    "NS": ["cody.ns.cloudflare.com", "fatima.ns.cloudflare.com"]
```

Comment:
We no longer support support new NS Records: 
![image](https://github.com/user-attachments/assets/fc3819ba-2215-40b3-be6e-57b3b6c18a98)